taverna-scufl2-cwl/workflow.t2flow.txt

```diff
@@ -0,0 +1,10 @@
+WorkflowBundle 'bundle1'
```

Comment:
I don't know if the workflow bundle structure format supports `#` comments for Apache license headers, but if not we will need to add an apache-rat exclude of this file to pom.xml  (there should be some already)